.eslintrc.js

```diff
@@ -4,7 +4,9 @@
     jest: true
   },
   rules: {
-    'no-proto': 0
-  },
+    'no-proto': 0,
+   'comma-dangle': 0,
+   'function-paren-newline': 0,
```

Comment:
Disabling 'function-paren-newline' might lead to readability issues in your code. Consider setting it to 'consistent' or another appropriate value to enhance code readability.